datasource/src/main/scala/quasar/physical/s3/S3DataSource.scala

```diff
@@ -52,12 +52,8 @@ final class S3DataSource[F[_]: Effect: Timer: MonadResourceErr](
       case Root =>
         Stream.empty.covaryAll[F, Data].pure[F]
       case Leaf(file) =>
-        impl.evaluate[F](config.parsing, client, config.bucket, file, S3DataSource.signRequest(config)) map {
+        impl.evaluate[F](config.parsing, client, config.bucket, file, signRequest(config)) map {
           case None => Stream.empty
-          /* In http4s, the type of streaming results is the same as
-           every other effectful operation. However,
-           LightweightDatasourceModule forces us to separate the types,
-           so we need to translate */
```

Comment:
Did this comment become obsolete or no longer relevant? Just wondering if this is useful for people browsing through this source looking to develop their own connector.

Reply:
It's no longer relevant since newer versions of Quasar use the same `F[_]` for both.